<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank Wire Transfer Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Bank Wire Transfer Simulator</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="accountFrom">
      <property name="placeholderText">
       <string>From Account ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="accountTo">
      <property name="placeholderText">
       <string>To Account ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="amount">
      <property name="placeholderText">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="transferButton">
      <property name="text">
       <string>Transfer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
